EIPS/eip-1.md

```diff
@@ -106,7 +106,7 @@ Each EIP should have the following parts:
 - Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale should discuss important objections or concerns raised during discussion around the EIP.
 - Backwards Compatibility *(optional)* - All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their consequences. The EIP must explain how the author proposes to deal with these incompatibilities. This section may be omitted if the proposal does not introduce any backwards incompatibilities, but this section must be included if backward incompatibilities exist.
 - Test Cases *(optional)* - Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Tests should either be inlined in the EIP as data (such as input/expected output pairs, or included in `../assets/eip-###/<filename>`. This section may be omitted for non-Core proposals.
-- Reference Implementation *(optional)* - An optional section that contains a reference/example implementation that people can use to assist in understanding or implementing this specification. This section may be omitted for all EIPs.
+- Reference Implementation *(optional)* - An optional section that contains a reference/example implementation that people can use to assist in understanding or implementing this specification. For ERC to advance to `Review` status, at least one reference implementation SHOULD be provided. For other EIPs this can be omitted. 
```

Comment:
For "this section may be omitted in any proposal.",

My understanding is that our consensus that we make it a strong suggestion for ERC to provide reference implementation. Some (me and gcolvin@) even think it's good to require them and only you explicitly say you feel negative about making a mandatory but ok with it being a strong recommendation.

And hence I prefer 
```
+ For other EIPs this can be omitted. 
```
over

```
- but this section may be omitted in any proposal.
```

Otherwise the wording become confusing.

For "status"

I think I prefer it to be added to a specific status, and having it at `Review` was what we discussed and agreed on the EIPIP meeting. its ok that we don't enforce it in the EIPW for the moment.